README.md

```diff
@@ -592,6 +592,32 @@ If you find that these options are not working, you can manually modify the keyb
 | <kbd>a</kbd> | `explorer.newFile`    |
 | <kbd>A</kbd> | `explorer.newFolder`  |
 
+### Hover widget manipulation
+
+> 💡 See [Keybindings help](#keybindings-help) to see all defined shortcuts and their documentation.
+
+You can add this to your keybindings.json: When hover is invisible, K is sent to nvim(show hover); when hover is
+visible, press K again to focus the hover widget.
+
+```json
```

Comment:
Should we just use this as default?

Reply:
https://github.com/vscode-neovim/vscode-neovim/issues/1681#issuecomment-1843839216

Reply:
Sure, it may not be needed, but we want to provide as complete and intuitive defaults as possible. If this is a natural and logical extension of the VSCode interface, manual configuration shouldn't be needed.

Reply:
I have no problem with it, most users should agree with this binding.